test/functional/api/vim_spec.lua

```diff
@@ -1036,5 +1036,33 @@ describe('api', function()
       eq(expected, nvim("list_uis"))
     end)
   end)
-
+  describe('nvim_get_keyboard_mods_state', function()
+    it('on linux either xlib11 is available or not', function()
+      if os_name() == 'unix' then
+        local status, rv = pcall(nvim, "get_keyboard_mods_state")
+        if status == true then
```

Comment:
Never check boolean variables for equality to something in conditions, they are boolean precisely for them to be put in condition directly.

Reply:
OKs